anaconda_linter/lint/__init__.py

```diff
@@ -86,9 +86,11 @@
 import inspect
 import logging
 import pkgutil
+from collections import defaultdict
+from dataclasses import dataclass, field
 from enum import IntEnum
 from pathlib import Path
-from typing import Any, Final, NamedTuple, Optional, Tuple, Union
+from typing import Any, Final, List, NamedTuple, Optional, Tuple, Union
```

Comment:
```suggestion
from typing import Any, Final, NamedTuple, Optional, Union
```
We should be refraining from using the capitalized versions of the common types

Reply:
(In favor of `list[]` and `tuple[]`)